documentation/status-logging.md

```diff
@@ -0,0 +1,55 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging through you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
```

Comment:
I don't think I'd document the float.  And I'd set the examples up to have a realistic value that a real-world user might actually use, like 30 mins or an hour or 2 hours.